case-server/src/main/java/com/powsybl/caseserver/CaseController.java

```diff
@@ -129,6 +131,18 @@ public ResponseEntity<UUID> importPublicCase(@RequestParam("file") MultipartFile
         return ResponseEntity.ok().body(caseUuid);
     }
 
+    @PostMapping(value = "/cases")
+    @Operation(summary = "duplicate a case")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The case has been duplicated"),
+            @ApiResponse(responseCode = "403", description = "An error occured during the case file duplication"),
+            @ApiResponse(responseCode = "404", description = "Parent case not found")})
+    @SuppressWarnings("javasecurity:S5145")
```

Comment:
why ?

Reply:
I carried it from the other creation methods, from what i gathered this prevents a false positive returned by sonar in connection with log4j. 
In this case it doesn't seem to be a problem to remove it, I think it's because no file is manipulated in this method unlike the other creation methods